### DISCUSSION TITLE
First time Calibration

### DISCUSSION THREAD

**Discussion by JalfasV2** [Netflix/OpenVPCal#58] [General]
Hello, 

So it is my first time using OpenVPCal. I managed to get all the way through and got very good results according to the software. Still, there is only one patch that seemed odd when applying the LUT to the wall. May be is supposed to be like this but I just wanted to make sure that this kind of result is normal. Here I attach the pictures of precalibrating and postcalibrating. It seems as if some coloured squares are missing. Is this normal or did I made a mistake somewhere? I would love a second opinion on this.

It is worth mentioning I am working with the Disgusie system, using a VX4+. My processor is a MX40Pro from Novastar and everything is set to PQ Rec2020. The camera I used is a RED Dragon X 5k. 

![Precalibration](https://github.com/user-attachments/assets/7ba517b8-3b1f-4cfd-8f3e-e9dbe91a09b6)

![PosCalibration](https://github.com/user-attachments/assets/76a30714-0370-4df3-ae95-e085daa1903f)

**Comment by davisadam10** [comment]
Looks like the LUTS where applied in Disguise without setting the input and output transforms on the LUT, also disguise pre OCIO needs to have the LUTS exported with the acescct options in OpenVPCal to work around some issues.

Disguise using OCIO removes this complexity
